src/app/index.ts

```diff
@@ -260,14 +221,20 @@ module.exports = yo.extend({
       this.project.projectInternalName = _.kebabCase(this.project.name);
       this.project.projectDisplayName = this.project.name;
       this.project.projectId = uuid();
-      this.project.hostInternalName = _.toLower(this.project.host);
-
-      if (this.project.folder) {
-        this.destinationRoot(this.project.projectInternalName);
+      if (this.project.projectType !== excelFunctions){
+        this.project.hostInternalName = this.project.host;
       }
+      else {
+        this.project.hostInternalName = excelFunctions;
+      }      
+      this.destinationRoot(this.project.folder);
+
+      /** Check to to see if destination folder already exists. If so, we will exit and prompt the user to provide
+      a different project name or output folder */
+      this._projectFolderExists();

```

Comment:
This should really be whether the folder is not empty. If not empty, prompt the user for another location. If empty or if does not exist, then it is OK to create/use it.